.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,20 @@
+name: auto-merge
+
+on:
+  pull_request_target:
+    types:
+    - labeled
+
+jobs:
+  auto-approve:
+    name: Auto-approve Konflux updates
+    runs-on: ubuntu-latest
+    if: github.actor == 'red-hat-konflux[bot]' && github.event.label.name == 'auto-approve'
+    steps:
+    - env:
+        GH_TOKEN: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
+      run: |
+        set -euo pipefail
```

Comment:
There are no `|` pipes and no environment variables in the below single-command script to justify the use of `set -euo pipefail`. I'm ok if you keep it because that's consistent with other our scripts but I'd note that we have many steps in GHA workflows where this is absent.

Also related:
- https://docs.github.com/en/enterprise-cloud@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsshell
- https://docs.github.com/en/enterprise-cloud@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsshell